containers/package-builder/checkout-repositories-version.sh.j2

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -e
+
+directory=$1
+repository=$2
```

Comment:
Please reverse the order of these parameters. The script is basically a wrapper over `git clone` and the order in clone is `<repository> <directory>`. It will make it easier to remember.

Reply:
Done.